pkg/registry/apis/dashboard/legacy/README.md

```diff
@@ -0,0 +1,15 @@
+This implements a ResourceServer backed by the existing dashboard SQL tables.
+
+There are a few oddities worth noting.  This is not a totally accurate implementation,
+but it is good enough to drive the UI needs and let kubectl list work!
+
+1. The resourceVersion is the dashboard version
+  - each resource starts at 1 and increases
+  - there are duplicate resourceVersions!
```

Comment:
we could use dashboard id * 1000000 + dashboard version if we want to avoid duplicates